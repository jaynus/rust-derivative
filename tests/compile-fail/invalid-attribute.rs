#[cfg(feature = "use_core")]
extern crate core;

#[macro_use]
extern crate derivative;

#[derive(Derivative)]
#[derivative(Clone = not_a_string)]
struct Foo;

#[derive(Derivative)]
#[derivative(Clone = 1+2)]
struct Bar;

fn main() {}