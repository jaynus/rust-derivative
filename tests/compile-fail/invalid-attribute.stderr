error: invalid attribute: expected literal
 --> $DIR/invalid-attribute.rs:8:22
  |
8 | #[derivative(Clone = not_a_string)]
  |                      ^^^^^^^^^^^^

error: invalid attribute: expected `,`
  --> $DIR/invalid-attribute.rs:12:23
   |
12 | #[derivative(Clone = 1+2)]
   |                       ^
